reference/nats-protocol/nats-protocol/README.md

```diff
@@ -162,15 +164,54 @@ where:
 
 To publish the ASCII string message payload "Hello NATS!" to subject FOO:
 
-`PUB FOO 11\r\nHello NATS!\r`
+`PUB FOO 11␍␊Hello NATS!␍␊`
 
 To publish a request message "Knock Knock" to subject FRONT.DOOR with reply subject INBOX.22:
 
-`PUB FRONT.DOOR INBOX.22 11\r\nKnock Knock\r`
+`PUB FRONT.DOOR INBOX.22 11␍␊Knock Knock␍␊`
 
 To publish an empty message to subject NOTIFY:
 
-`PUB NOTIFY 0\r\n\r`
+`PUB NOTIFY 0␍␊␍␊`
+
+## HPUB
+
+### Description
+
+The `HPUB` message is the same as `PUB` but extends the message payload to include NATS headers. Note that the payload itself is optional. To omit the payload, set the total message size equal the size of the headers. Note that the trailing CR+LF is still required.
```

Comment:
```suggestion
The `HPUB` message is the same as `PUB` but extends the message payload to include NATS headers. Note that the payload itself is optional. To omit the payload, set the total message size equal to the size of the headers. Note that the trailing CR+LF is still required.
```

Reply:
I think I added this correction before. In case it was overlooked, just calling it out again.. or if the current wording is grammatically correct then that is fine too. It just reads a bit weird for me.